network/p2p/capabilities.go

```diff
@@ -131,17 +141,17 @@ func (c *CapabilitiesDiscovery) AdvertiseCapabilities(capabilities ...Capability
 					ttl, err0 := c.advertise(c.dht.Context(), string(capa))
 					if err0 != nil {
 						err = err0
-						c.log.Errorf("failed to advertise for capability %s: %v", capa, err0)
+						c.log.Warnf("failed to advertise for capability %s: %v", capa, err0)
 						break
 					}
 					if ttl < advertisementInterval {
 						advertisementInterval = ttl
 					}
 					c.log.Infof("advertised capability %s", capa)
 				}
-				// If we failed to advertise, retry every 10 seconds until successful
+				// If we failed to advertise, retry every 100 seconds until successful
```

Comment:
The comment before the function needs adjustment for 100 instead of 10.

Reply:
Do we think the ttl for different capabilities caries a lot? If so, it might be worth managing their advertisments separately, with len(capabilities) goroutines, so each uses the approriate ttl.

Reply:
updated the comment.

> Do we think the ttl for different capabilities varies a lot? 

IMO they are all the same importance so I thought shuffling would look a bit cleaner than each own goroutine for each capability but sure can re-do if you feel strongly

Reply:
I was imagining that one capability might have a ttl of 10 seconds, while another might have a ttl of 1 day. In that case, we re-advertise all of them every 10 seconds because it's the minimum.

But if they are all about the same, this is fine.